frontend/src/app/api.service.ts

```diff
@@ -75,6 +75,20 @@ export class ApiService {
     );
   }
 
+  /**
+   * @description This method makes an http call and fetches the data for events from the specified file.
+   *
+   * @param string filename The name of the file containing the events data
+   * @param object filters Filters for the data. This is an object with fields: countries, device, fromDate, toDate, timePeriod
+   */
+  getEventsData(filename, filters): Observable<object> {
+    const params = this.buildFilter(filters, new HttpParams());
+    return this.http.get(
+      environment.apiRest + 'events/' + filename,
```

Comment:
Please only merge this PR after the backend part of this call is already merged, so the repo won't have an inconsistent state.